--- ecasound-1.9dev1/configure.in~	Wed Feb  7 04:36:48 2001
+++ ecasound-1.9dev1/configure.in	Sat Apr 21 09:46:51 2001
@@ -70,14 +70,6 @@
 
 dnl ------------------------------------------------------------------
 
-dnl ---
-dnl Set CXXFLAGS.
-dnl ---
-CXXFLAGS="-O2 -D_REENTRANT -DNDEBUG -ffast-math -fstrict-aliasing -funroll-loops"
-dnl CXXFLAGS="-O2 -D_REENTRANT -fomit-frame-pointer -finline-functions -ffast-math -funroll-loops -Wall -Wstrict-prototypes -pedantic -fstrict-aliasing"
-
-dnl ------------------------------------------------------------------
-
 AC_ARG_ENABLE(debug,
 [  --enable-debug		Enable debugging (default = no)],
 include_debug=yes)
