--- ecasound-2.0.2.new/ecasound/Makefile.am~	Fri Feb  9 03:09:29 2001
+++ ecasound-2.0.2.new/ecasound/Makefile.am	Mon Sep 10 23:59:09 2001
@@ -1,23 +1,9 @@
-if INCLUDE_DEBUG
-INSTALL_STRIP_FLAG =
-else
-INSTALL_STRIP_FLAG = -s
-endif
-
 if SYSTEM_READLINE
 INCLUDES = -I$(readline_includes) -I$(srcdir) -I$(top_srcdir) -I$(top_srcdir)/libecasound -I$(top_srcdir)/kvutils
 else
 INCLUDES = -I$(top_srcdir)/readline-4.0 -I$(srcdir) -I$(top_srcdir) -I$(top_srcdir)/libecasound -I$(top_srcdir)/kvutils
 endif
 
-if USE_NCURSES
-termcap_library = -lncurses
-endif
-
-if USE_TERMCAP
-termcap_library = -ltermcap
-endif
-
 if INCLUDE_DEBUG
 bin_PROGRAMS = ecasound_debug
 else
@@ -28,19 +14,19 @@
 ecasound_debug_SOURCES = $(ecasound_SOURCES)
 
 if SYSTEM_READLINE
-ecasound_LDADD = $(termcap_library) \
+ecasound_LDADD = \
 		-L$(top_builddir)/libecasound/.libs -lecasound \
 		-L$(top_builddir)/kvutils/.libs -lkvutils
-ecasound_debug_LDADD = $(termcap_library) \
+ecasound_debug_LDADD = \
 		-L$(top_builddir)/libecasound/.libs -lecasound_debug \
 		-L$(top_builddir)/kvutils/.libs -lkvutils
 else
 ecasound_LDADD = $(top_builddir)/readline-4.0/libreadline.a \
-		$(top_builddir)/readline-4.0/libhistory.a $(termcap_library) \
+		$(top_builddir)/readline-4.0/libhistory.a \
 		$(top_builddir)/libecasound/libecasound.la \
 		-L$(top_builddir)/kvutils/.libs -lkvutils
 ecasound_debug_LDADD = $(top_builddir)/readline-4.0/libreadline.a \
-		$(top_builddir)/readline-4.0/libhistory.a $(termcap_library) \
+		$(top_builddir)/readline-4.0/libhistory.a \
 	 	$(top_builddir)/libecasound/libecasound_debug.la \
 		-L$(top_builddir)/kvutils/.libs -lkvutils
 endif
--- ecasound-2.0.2.new/kvutils/Makefile.am~	Mon Sep 10 23:54:50 2001
+++ ecasound-2.0.2.new/kvutils/Makefile.am	Tue Sep 11 00:01:27 2001
@@ -29,11 +29,9 @@
 libkvutils_la_SOURCES = $(kvutil_sources) $(kvutil_headers)
 libkvutils_la_LDFLAGS = -version-info 2:0:0
 
-DIST_COMMON = 	README \
+EXTRA_DIST = 	README \
 		COPYING \
-		INSTALL \
-		Makefile.in \
-		Makefile.am
+		INSTALL
 
 noinst_HEADERS = $(kvutil_headers)
 
