--- ecasound-2.1.0/configure.in.orig	Sun Oct  7 17:39:44 2001
+++ ecasound-2.1.0/configure.in	Mon Feb  4 04:13:30 2002
@@ -71,15 +71,6 @@
 
 dnl ------------------------------------------------------------------
 
-dnl ---
-dnl Set CXXFLAGS.
-dnl ---
-CXXFLAGS="-O2 -D_REENTRANT -DNDEBUG -ffast-math -fstrict-aliasing -funroll-loops"
-CFLAGS="$CXXFLAGS"
-dnl --- CXXFLAGS="-O2 -D_REENTRANT -fomit-frame-pointer -finline-functions -ffast-math -funroll-loops -Wall -Wstrict-prototypes -pedantic -fstrict-aliasing"
-
-dnl ------------------------------------------------------------------
-
 AC_MSG_CHECKING(whether to enable debugging)
 AC_ARG_ENABLE(debug,
 [  --enable-debug		Enable debugging (default = no)],
@@ -189,25 +180,25 @@
 dnl ---
 dnl Check for ncurses/termcap
 dnl ---
-AC_CHECK_LIB(ncurses, main, termcap_library=ncurses,
-	AC_CHECK_LIB(termcap, main, termcap_library=termcap,
+AC_CHECK_LIB(ncurses, main, [termcap_library=ncurses],
+	AC_CHECK_LIB(termcap, main, [termcap_library=termcap],
 		AC_MSG_ERROR([** neither termcap nor ncurses library is installed (needed by the console mode user interface) **])))
 AC_ARG_ENABLE(ncurses,
 [  --disable-ncurses	  	Disable ncurses (default = no)],
-termcap_library=none)
-AM_CONDITIONAL(USE_NCURSES, test $termcap_library = ncurses)
-AM_CONDITIONAL(USE_TERMCAP, test $termcap_library = termcap)
+[termcap_library=none])
+AM_CONDITIONAL(USE_NCURSES, [test $termcap_library = ncurses])
+AM_CONDITIONAL(USE_TERMCAP, [test $termcap_library = termcap])
 if test x$termcap_library = xtermcap; then
 	AC_DEFINE(USE_TERMCAP)
 fi
 if test x$termcap_library = xncurses; then
 	AC_CHECK_HEADER(curses.h,
-	 	AC_CHECK_HEADER(term.h,, AC_MSG_ERROR([*** curses headers not installed ***])),
-    	  		AC_CHECK_HEADER(ncurses/curses.h, [ 
-           			AC_CHECK_HEADER(ncurses/term.h,, AC_MSG_ERROR([*** curses headers not installed ***]))
+	 	[AC_CHECK_HEADER(term.h, [], AC_MSG_ERROR([*** curses headers not installed ***]))],
+    	  		[AC_CHECK_HEADER(ncurses/curses.h, [ 
+           			AC_CHECK_HEADER(ncurses/term.h, [], AC_MSG_ERROR([*** curses headers not installed ***]))
              			CXXFLAGS="-DHAVE_NCURSES_CURSES_H $CXXFLAGS"
       	     			],
-      	   			AC_MSG_ERROR([*** curses headers not installed ***]))
+      	   			AC_MSG_ERROR([*** curses headers not installed ***]))]
         )
 	AC_DEFINE(USE_NCURSES)
 fi
@@ -385,19 +376,19 @@
 dnl Check for readline.h or readline/readline.h (same with history.h)
 dnl and for readline and history libraries
 dnl
-AM_CONDITIONAL(SYSTEM_READLINE, test x$enable_sysreadline = xyes)
+AM_CONDITIONAL(SYSTEM_READLINE, [test x$enable_sysreadline = xyes])
 if test x$enable_sysreadline = xyes; then
-  AC_CHECK_HEADER(readline.h, readline_includes=/usr/include, 
-    AC_CHECK_HEADER(readline/readline.h,
-      readline_includes=/usr/include/readline,
-      AC_MSG_ERROR([*** readline headers not installed ***]))
+  AC_CHECK_HEADER(readline.h, [readline_includes=/usr/include],
+    [AC_CHECK_HEADER(readline/readline.h,
+      [readline_includes=/usr/include/readline],
+      AC_MSG_ERROR([*** readline headers not installed ***]))]
   )
-  AC_CHECK_HEADER(history.h,,
-    AC_CHECK_HEADER(readline/history.h,,
-      AC_MSG_ERROR([*** readline history headers not installed ***]))
+  AC_CHECK_HEADER(history.h, [],
+    [AC_CHECK_HEADER(readline/history.h, [],
+      AC_MSG_ERROR([*** readline history headers not installed ***]))]
   )
-  AC_CHECK_LIB(readline, main,, AC_MSG_ERROR([*** readline support not installed ***]))
-  AC_CHECK_LIB(history, main,, AC_MSG_ERROR([*** readline history support not installed ***]))
+  AC_CHECK_LIB(readline, main, [], AC_MSG_ERROR([*** readline support not installed ***]))
+  AC_CHECK_LIB(history, main, [], AC_MSG_ERROR([*** readline history support not installed ***]))
   AC_SUBST(readline_includes)
 else
   AC_CONFIG_SUBDIRS(readline-4.0)
